client/src/static/image-urls.ts

```diff
@@ -24,3 +24,4 @@ export const Option = BASE_URL + 'option.svg';
 export const Xbox = BASE_URL + 'xbox.svg';
 export const CheckActive = BASE_URL + 'checkActive.svg';
 export const CheckNonActive = BASE_URL + 'checkNonActive.svg';
+export const TRASHCAN = BASE_URL + 'trashcan.svg';
```

Comment:
상수 명은 일관성을 지켜주는 것이 좋을 것 같습니다

```suggestion
export const Xbox = BASE_URL + 'xbox.svg';
export const CheckActive = BASE_URL + 'checkActive.svg';
export const CheckNonActive = BASE_URL + 'checkNonActive.svg';
export const TrashCan = BASE_URL + 'trashcan.svg';
```

Reply:
옙 반영하겠습니다!!